core/pom.xml

```diff
@@ -89,6 +89,31 @@
             <version>${project.parent.version}</version>
             <scope>test</scope>
         </dependency>
+
+        <dependency>
+            <groupId>org.mockito</groupId>
+            <artifactId>mockito-all</artifactId>
+            <version>1.10.19</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.powermock</groupId>
+            <artifactId>powermock-api-mockito</artifactId>
```

Comment:
The dependencies listed at:
https://github.com/diffblue/platform/blob/develop/docs/user/topics/executing-diffblue-tests.md
also include powermock-api-mockito-common and has deeptestutils at 1.5.0.
Is that document out of date?

Reply:
Yes. `power-mock-api-mockito-common` is not necessary, and the latest version of deeptestutils is 1.7.1